src/otel/logger.ts

```diff
@@ -172,6 +172,59 @@ export function patchConsole(
 	_patch.info = (...args: unknown[]) => {
 		delegate.info(args[0] as string, ...args.slice(1));
 	};
+	
+	_patch.dir = (...args: unknown[]) => {
```

Comment:
if the original console is getting spread into _patch, isn't this not doing anything?